lib/google/cloud/gemserver/cli/request.rb

```diff
@@ -54,8 +54,6 @@ def initialize url = nil, proj_name = nil
           #
           # @param [String] permissions The permissions the generated key will
           # have (read, write, or both). Optional.
-          #
-          # @return [String]
```

Comment:
It depends on the response, e.g. Net::HTTPOk on a 200, Net::HTTPNotFound for a 404, etc. Is there a good way to represent that?